tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -417,9 +417,9 @@ internal SpanContext CreateSpanContext(ISpanContext parent = null, string servic
             return spanContext;
         }
 
-        internal Scope StartActiveWithTags(string operationName, ISpanContext parent = null, string serviceName = null, DateTimeOffset? startTime = null, bool ignoreActiveScope = false, bool finishOnClose = true, ITags tags = null, ulong? spanId = null)
+        internal Scope StartActiveWithTags(string operationName, ISpanContext parent = null, string serviceName = null, DateTimeOffset? startTime = null, bool ignoreActiveScope = false, bool finishOnClose = true, ITags tags = null)
         {
-            var span = StartSpan(operationName, tags, parent, serviceName, startTime, ignoreActiveScope, spanId);
+            var span = StartSpan(operationName, tags, parent, serviceName, startTime, ignoreActiveScope);
```

Comment:
This `StartSpan` method takes an optional `ulong? spanId = null` too, but I think this was the only usage, so we should probably remove that too.

Reply:
`ScopeFactory.CreateInactiveOutboundHttpSpan` still uses the spanId/traceId arguments for `StartSpan`, to implement the weird `BeginGetRequestStream`/`BeginGetResponse` instrumentation

Reply:
ah, I think it's because I was maybe on the `release/2.0` branch, and that's not rebased on top of `master` yet, thought there was an unused overload. Nevermind!